sidebars.ts

```diff
@@ -203,6 +203,16 @@ module.exports = {
                         'send-data/opentelemetry-collector/remote-management/source-templates/nginx/changelog',
                       ]
                     },
+                    //{
+                      //type: 'category',
+                      //label: 'PostgreSQL',
+                      //collapsible: true,
+                      //collapsed: true,
+                      //link: {type: 'doc', id: 'send-data/opentelemetry-collector/remote-management/source-templates/postgresql/index'},
+                      //items:[
+                        //'send-data/opentelemetry-collector/remote-management/source-templates/postgresql/changelog',
+                      //]
+                    //},
```

Comment:
@sumoanema since you've labeled the docs as beta, I've added them to the sidebars and commented them out. When you're ready, you can uncomment these lines.

Reply:
Since OTRM has gone GA these ST are not beta anymore. Have uncommented the entry in sidebar.js and added two more - mysql and elasticsearch doc for which are added to this PR.

Reply:
Since these are GA, you'll need to remove beta code, which looks like this @sumoanema 

```
<head>
  <meta name="robots" content="noindex" />
</head>

<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
```

Reply:
Thanks @kimsauce for pointing that out. Have merged your suggestions.